index.bs

```diff
@@ -982,42 +950,41 @@ WebGL Context Compatiblity {#contextcompatibility}
 
 <pre class="idl">
 partial dictionary WebGLContextAttributes {
-    XRDevice compatibleXRDevice = null;
```

Comment:
Etiquette question: given that these are WebGL changes, can they actually be specified in this document or do they need to be done differently?

Reply:
There are existing scenarios where one spec implies additions or related functionality to another one. (That's more-or-less why we have the `partial` keyword.) So I think this is appropriately scoped. For example, search for "WebGL" here: https://www.w3.org/TR/2017/WD-html52-20170718/single-page.html